agent_api_rest/postman/ssi-agent.postman_collection.json

```diff
@@ -1981,6 +1982,23 @@
 					"item": [
 						{
 							"name": "Create a new Template",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"if (pm.response.code === 201) {",
+											"    const response = pm.response.json();",
+											"    pm.collectionVariables.set(\"template_id\", response.id);",
+											"    console.log(\"Saved template_id: \" + response.id);",
```

Comment:
Here as well as in the Request Body for "Duplicate Template" you use `template_id` but for the other endpoints we use `TEMPLATE_ID`. Is that deliberate or can we use `TEMPLATE_ID` for everything?